src/ConstrainedPhotometryModel.cc

```diff
@@ -188,11 +188,10 @@ std::shared_ptr<afw::image::PhotoCalib> ConstrainedPhotometryModel::toPhotoCalib
     ast::ZoomMap zoomMap(1, mapping->getChipMapping()->getParameters()[0]);
 
     // Now stitch them all together.
-    auto transform =
-            afw::geom::TransformPoint2ToGeneric(pixToFocal->getFrameSet()->then(chebyMap).then(zoomMap));
+    auto transform = pixToFocal->then(afw::geom::TransformPoint2ToGeneric(chebyMap.then(zoomMap)));
```

Comment:
Why the re-ordering of `then` here? I see the removal of `getFrameSet()`, but I'm not sure I understand the logic of the rest: we want this to be a `TransformPoint2ToGeneric` in the end, don't we?

Reply:
I rewrote it in a way that @parejkoj preferred